src/www/ui/change-license-processPost.php

```diff
@@ -95,20 +95,31 @@ function Output()
     $decisionMark = @$_POST['decisionMark'];
     if(!empty($decisionMark))
     {
-      $itemTableName = $this->uploadDao->getUploadtreeTableName($itemId);
-      /** @var ItemTreeBounds */
-      $itemTreeBounds = $this->uploadDao->getItemTreeBounds($itemId,$itemTableName);
-      $errMsg = $this->clearingDao->markDirectoryAsIrrelevant($itemTreeBounds,$groupId,$userId);
-      if (empty($errMsg))
-      {
-        return new JsonResponse(array('result'=>'success'));
+      if(!is_array($itemId)) {
+        $this->doMarkIrrelevant($itemId, $groupId, $userId);
+      }else{
+        foreach($itemId as $uploadTreeId){
+          $this->doMarkIrrelevant($uploadTreeId, $groupId, $userId);        
+        }
       }
-      return $this->errorJson($errMsg,$errMsg);
+      return new JsonResponse(array('result'=>'success'));
     }
 
     return $this->doEdit($userId,$groupId,$itemId);
   }
 
+  function doMarkIrrelevant($itemId, $groupId, $userId)
+  {
+    $itemTableName = $this->uploadDao->getUploadtreeTableName($itemId);
+    /** @var ItemTreeBounds */
+    $itemTreeBounds = $this->uploadDao->getItemTreeBounds($itemId,$itemTableName);
+    $errMsg = $this->clearingDao->markDirectoryAsIrrelevant($itemTreeBounds,$groupId,$userId);
+    if(!empty($errMsg)){
+      return $this->errorJson($errMsg,$errMsg);
+    }
+    return $errMsg;
```

Comment:
At this point `$errMsg` is always empty. Why not return something useful or a default value?

Reply:
Previously here was `return new JsonResponse(array('result'=>'success'));` used

Reply:
You are right $errMsg will be empty. but there is no additional case handled here so couldn't return a default value. or shall i do just "return;" ??

Reply:
maybe returning `""` would be better.

Reply:
sure ok.